packages/builder/src/components/userInterface/Colorpicker/components/Slider.svelte

```diff
@@ -11,11 +11,17 @@
   let sliderWidth = 0
   let upperLimit = type === "hue" ? 360 : 1
   let incrementFactor = type === "hue" ? 1 : 0.01
+  let cursor = "grab"
 
   const isWithinLimit = value => value >= 0 && value <= upperLimit
 
-  function onSliderChange(mouseX, isDrag = false) {
+  function onSliderChange({ mouseX }, isDrag = false) {
     const { left, width } = slider.getBoundingClientRect()
+
+    if (isDrag && cursor !== "grabbing") {
```

Comment:
Does the `:active` pseudo class allow you to do this?

Reply:
It does indeed. Good man 👍